decentraland/redux/reducers.js

```diff
@@ -50,10 +51,21 @@ function assetsForSale(state = { list: [], loadingInProgress: true }, action) {
   }
 }
 
+function myAssets(state = { list: [] }, action) {
+  const { type, myAsset } = action;
+  switch (type) {
+    case ADD_TO_MY_ASSETS_LIST:
+      return { ...state, list: [...state.list, myAsset] };
```

Comment:
Maybe put it at the front of the list rather than the end so it shows up on top?

Reply:
Done

Reply:
Thanks! Resolving